tests/drivers/build_all/power_domain/prj.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_TEST=y
+CONFIG_TEST_USERSPACE=y
+CONFIG_GPIO=y
+CONFIG_POWER_DOMAIN=y
+CONFIG_PM_DEVICE=y
```

Comment:
`PM_DEVICE` and `PM_DEVICE_RUNTIME` should not be added here, as they are enabled in the `drivers.power_domain.build.pm` testcase.

Reply:
Thank you for your check.
I've fixed it.
"power_domain,gpio_monitor" completely depends on PM_DEVICE, so I also fixed Kconfig.